scripts/cli.sh

```diff
@@ -18,6 +18,14 @@ setup_cli () {
     bash scripts/ci/setupCLIConfig.sh
 }
 
+# Set the path ENV for the global config
+# Because `scripts/ci/setupCLIConfig.sh` gets called with `bash` and not `source` || `.`
+# Any envs exported from the script are not loaded in this session
+# Even though line 44 of `setupCLIConfig.sh` exports the `KEG_GLOBAL_CONFIG`, we can't access it
+# Instead of setting this ENV here, we could update line 18 of this file
+# to use `source` instead of `bash`
+export KEG_GLOBAL_CONFIG=$ACTION_WORKSPACE/keg-cli/.kegConfig/cli.config.json
```

Comment:
This is the reason the `globalConfig` was not being set properly running this action.
* The main issue is on `line 18`, it's using `bash` instead of `source` or `.` to run the `setupCLIConfig.sh` file
* Within the `setupCLIConfig.sh` file, the `KEG_GLOBAL_CONFIG` env gets set, but because `bash` is being used on `line 18`, it creates a new sub-session to run that script 
* So the `KEG_GLOBAL_CONFIG` env does not exist within this session

So we can either set it ourselves, or update line 18 to use `source` instead of `bash`